bfvmm/include/vmcs/vmcs_intel_x64_state.h

```diff
@@ -277,6 +279,19 @@ class vmcs_intel_x64_state
     virtual void set_ia32_gs_base_msr(intel_x64::msrs::value_type val)
     { (void) val; }
 
+    virtual integer_pointer rip() const
+    { return 0; }
+    virtual integer_pointer rsp() const
+    { return 0; }
+
+    virtual void set_rip(integer_pointer val) const
+    { (void) val; }
+    virtual void set_rsp(integer_pointer val) const
+    { (void) val; }
+
+    virtual bool is_guest()
```

Comment:
No change at the moment, I think we should strive for the APIs to be orthogonal to the collection of cases is all. Just something to keep in mind.